util/lib.py

```
@@ -342,41 +342,54 @@
-    logging.info(f"Preparing {o_path} for dashboard.")
-    
-    callstring = ['robot', 'merge', '-i', o_path]
-    
+def robot_prepare_ontology(
+    o_path,
+    o_out_path,
+    o_metrics_path,
+    base_iris,
+    make_base,
+    robot_prefixes=None,
+    robot_opts="-v"
```

Ruff

Missing type annotation for function argument `robot_opts`

(ANN001)